lib/ama_layout/version.rb

```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = '4.7.1'
+  VERSION = '5.0.0'
```

Comment:
Should this be a major version bump? We're not really making breaking changes, more so just adding a bunch of features?